src/config/Blocks.jsx

```diff
@@ -289,6 +289,14 @@ const blocksConfig = {
       },
     ],
     getAsyncData: getListingBlockAsyncData,
+    tocEntry: (blockData = {}, _) => {
```

Comment:
Please document the `tocEntry` setting here: https://6.dev-docs.plone.org/volto/blocks/settings.html